modules/aws/ssm.go

```diff
@@ -0,0 +1,67 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"github.com/stretchr/testify/require"
+)
+
+func GetParameter(t *testing.T, awsRegion string, keyName string) string {
```

Comment:
One last thing: could you add comments to these new methods explaining that they are reading parameters from SSM?

Reply:
@brikis98 Totally. Sorry forgot to do it initially. Should be good to go.